packages/repository-tests/src/index.ts

```diff
@@ -4,6 +4,7 @@
 // License text available at https://opensource.org/licenses/MIT
 
 export * from './crud-test-suite';
+export * from './transaction-test-suite';
```

Comment:
Transactions are CRUD specific, these tests are not going to work for KeyValue repositories.

Please move the new test file `transaction-test-suite` to `packages/repository-tests/src/crud` directory, where it will be automatically picked up by our infrastructure.

It looks like our developer documentation is insufficiently describing how to add new suites, please improve it to help the next person: https://github.com/strongloop/loopback-next/tree/master/packages/repository-tests#developer-guide

Please sync with @hacksparrow, he is working on this part of our docs too.